docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -89,6 +89,10 @@ You can add these Kubernetes annotations to specific Ingress objects to customiz
 |[nginx.ingress.kubernetes.io/influxdb-host](#influxdb)|string|
 |[nginx.ingress.kubernetes.io/influxdb-server-name](#influxdb)|string|
 |[nginx.ingress.kubernetes.io/use-regex](#use-regex)|bool|
+|[nginx.ingress.kubernetes.io/enable-modsecurity](#modsecurity)|"on" or "off|
```

Comment:
please use bool ("true" or "false") to be consistent with the other annotations

Reply:
@aledbf I chose to use "on" or "off" because of the templating. If I were to choose boolean then the `{{ if (and (not (empty $location.ModSecurity.Enable)) $all.Cfg.EnableModsecurity) }}` case would not go through if it was set to false(and it is default set to false). 

We want "off" and modSecurity module enabled to signify, that modsecurity will be off on a certain location. If we use boolean it is hard to enable this case.

Reply:
> I chose to use "on" or "off" because of the templating

I understand that be we need to be consistent. Please change to true/false

Reply:
not sure how I will be able to do the off condition :( I'll look into it, but doesn't seem workable.

Reply:
@aledbf ^

Reply:
What do you mean? https://gist.github.com/aledbf/e0aeba3a0c33c97c93d8787b23d86960

Reply:
That scenario makes sense, but it means that if modsecurity is enabled in the configmap it will not be enabled for all locations. Now to enable modsecurity it would only be scoped per location.

If that is ok, lmk and I will add more changes to the documentation.

@aledbf ^

Reply:
I will add the change, after thinking about it, makes sense to do ModSecurity per Ingress. 💯

Reply:
:+1:

Reply:
you need to remove the conditional load of the module `load_module /etc/nginx/modules/ngx_http_modsecurity_module.so;`

Reply:
Do I still need to remove the conditional. I think it still makes sense in this case. I updated the PR.

Reply:
Nevermind you are completely right :)